app/assets/stylesheets/application.scss

```diff
@@ -15,28 +15,74 @@
 *= require foundation_and_overrides
 *= require_self
  */
+@import url(https://fonts.googleapis.com/css?family=Montaga);
+$lightgray: #ede0d5;
+$blue: #354a57;
 $borderbrown: #a52a2a;
 $backgroundorange: #ffffcc;
 $fontgreen: #008000;
+$fontblue: #94c9ff;
 
-.side-nav {
-  height: 100vh;
+
+.content {
+  padding-left: 20px;
+}
+.side-nav  {
+  background-color: #b94947;
+  padding: 1em;
+  color:$blue !important ;
+  h1, h2, h3, h4, h5 {
+    color:$blue !important ;
+  }
+
+}
+
+.searchbar {
+  display: inline-block;
+}
+
+.page-end {
+  font-size: 2em;
+}
+
+.explore-pic img {
+  border: 10px solid;
+  border-color: $blue ;
+}
+
+.search-page img{
+  border: 10px solid;
+  border-color: $blue ;
+}
+
+.body {
+  font-family: 'Montaga', serif;
+  color: #333;
 }
 
-.side-nav-border {
-  width: 110%;
+ul {
+
+  color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.